packages/vuetify/src/components/VTooltip/index.ts

```diff
@@ -1,4 +1 @@
-import VTooltip from './VTooltip'
-
-export { VTooltip }
-export default VTooltip
+export { default as VTooltip } from './VTooltip'
```

Comment:
```suggestion
export * from './VTooltip'
```

We talked about having this nomenclature change because we needed typing for VInput on another branch.

Reply:
I'll do this later in bulk for the whole framework.